courses/free-courses-id.md

```diff
@@ -396,6 +400,11 @@
 * [Tutorial Blockchain Bahasa Indonesia](https://www.youtube.com/playlist?list=PLH1gH0TmFBBhvZi4kEqU6kCjyv_y8qBae) - Ipung DEV Academy (YouTube)
 
 
+### Terraform
+
+* [Bajalar Terraform untuk Pemula](https://www.youtube.com/watch?v=Ce4Q1xLDNjE&list=PL4SGTPmSY0qngs44Ssc0RHO9h4fmZ9JUb) - Giri Kuncoro (YouTube)
```

Comment:
Please add playlist link instead of the video link

Reply:
Hello @shrn01, thanks for pointing these out. 
I have fixed it in my latest PR.